app/sanitizers/rendered_markdown_scrubber.rb

```diff
@@ -3,14 +3,14 @@ def initialize
     super
 
     self.tags = %w[
-      a abbr add aside b blockquote br button center cite code col colgroup dd del dl dt em em figcaption
-      h1 h2 h3 h4 h5 h6 hr i img kbd li mark ol p pre q rp rt ruby small source span strong sub sup table
+      a abbr add b blockquote br button center cite code col colgroup dd del dl dt em figcaption
+      h1 h2 h3 h4 h5 h6 hr img kbd li mark ol p pre q rp rt ruby small source span strong sub sup table
       tbody td tfoot th thead time tr u ul video
     ]
 
     self.attributes = %w[
-      alt colspan data-conversation data-lang data-no-instant data-url em height href id loop
-      name ref rel rowspan size span src start strong title type value width controls
+      alt colspan data-conversation data-lang data-no-instant data-url href id loop
+      name ref rel rowspan span src start title type value controls
```

Comment:
`<em>` and `<strong>` are not valid attributes. 🥴 

`<size>`, `<height>`, and `<width>` I think are a bit dangerous to allow people to change.